interface/main/calendar/add_edit_event.php

```diff
@@ -1242,13 +1310,19 @@ function find_available(extra) {
 	$cid=$_REQUEST["catid"];
 ?>
 		 <li <?php echo $normal;?>>
-		 <a href='add_edit_event.php?eid=<?php echo attr($eid);?>&startampm=<?php echo attr($startm);?>&starttimeh=<?php echo attr($starth);?>&userid=<?php echo attr($uid);?>&starttimem=<?php echo attr($starttm);?>&date=<?php echo attr($dt);?>&catid=<?php echo attr($cid);?>'>
+		 <a href='add_edit_event.php?startampm=<?php echo attr($startm);?>&starttimeh=<?php echo attr($starth);?>&userid=<?php echo attr($uid);?>&starttimem=<?php echo attr($starttm);?>&date=<?php echo attr($dt);?>&catid=<?php echo attr($cid);?>'>
 		 <?php echo xlt('Patient');?></a>
 		 </li>
 		 <li <?php echo $provider_class;?>>
-		 <a href='add_edit_event.php?prov=true&eid=<?php echo attr($eid);?>&startampm=<?php echo attr($startm);?>&starttimeh=<?php echo attr($starth);?>&userid=<?php echo attr($uid);?>&starttimem=<?php echo attr($starttm);?>&date=<?php echo attr($dt);?>&catid=<?php echo attr($cid);?>'>
```

Comment:
The problem imposed by retaining the eid:
If there is an existing groups event and we then click on this event, switch to patients tab, and save the event. The system will try to 'convert' the event to a patients event. This caused many issues - part of the old group parameters stayed and part were converted. To fix this is quite complicated and the simplest and cleanest solution was to prevent 'converting' one type of event to another (which I don't think should be allowed from a user's point of view - it will mess up the history). Besides for this fix, there should be no other consequences from the change.